src/app/core/services/si/si-core/intacct-connector.service.ts

```
@@ -42,6 +42,11 @@ export class IntacctConnectorService {
     return this.apiService.post('/workspaces/' + this.workspaceId + '/credentials/sage_intacct/', data);
   }
 
+  @Cacheable()
+  checkIntacctTokenHealth(workspaceId: string): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)